src/types/errors.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// @flow
+
+export type Error = {|
+    server_error_id?: string,
+    stack?: string,
+    message: string,
```

Comment:
I think there is a status_code missing here

Reply:
Yeah I had it in there as `status_code?: number` because I found some errors were being sent without it and then I second guessed myself due to so many optional params. I'll put it back in there!